ott/core/neuraldual.py

```diff
@@ -99,31 +102,30 @@ def __init__(
     # set optimizer and networks
     self.setup(rng, neural_f, neural_g, input_dim, optimizer_f, optimizer_g)
 
-  def setup(self, rng, neural_f, neural_g, input_dim, optimizer_f, optimizer_g):
+  def setup(
+      self, rng: jnp.ndarray, neural_f: icnn.ICNN, neural_g: icnn.ICNN,
+      input_dim: int, optimizer_f, optimizer_g
+  ) -> None:
     """Setup all components required to train the `NeuralDual`."""
```

Comment:
Here I guess we now mean `DualPotentials` ?

Reply:
Done as `Setup all components required to train the network.`.